src/web/app/pages-instructor/instructor-search-page/student-result-table/student-result-table.component.ts

```diff
@@ -24,6 +24,8 @@ export class StudentResultTableComponent {
 
   @Input() studentTables: SearchStudentsListRowTable[] = [];
   @Input() isActionButtonsEnabled: boolean = true;
+  @Input() isSearchTermsHighlighted: boolean = false;
+  @Input() searchedTerm: string = '';
 
```

Comment:
A search term is a single term, a user's search string can contain multiple search terms, I think you mean `searchString`